test/functional/page_objects/dashboard_page.js

```diff
@@ -169,8 +169,16 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
       await testSubjects.setValue('clonedDashboardTitle', title);
     }
 
-    async isDuplicateTitleWarningDisplayed() {
-      return await testSubjects.exists('titleDupicateWarnMsg');
+    /**
+     * Asserts that the duplicate title warning is either displayed or not displayed.
+     * @param { displayed: boolean }
+     */
+    async expectDuplicateTitleWarningDisplayed({ displayed }) {
+      if (displayed) {
+        await testSubjects.existOrFail('titleDupicateWarnMsg');
+      } else {
```

Comment:
testSubjects.existOrFail calls
testSubjects.exists which calls
find.existsByDisplayedByCssSelector which calls
find.exists which has a default 1 second timeout.  Is that appropriate for this case?

Reply:
it *should* work because of the outer retry in existOrFail.  The inner code will complete in 1 second, but it will retry `testSubjects.exists` for the default retry.try timeout which I believe is 60 seconds for `testSubjects.exists` to eventually return true.

I verified locally that `testSubjects.existOrFail` does do the right thing when it doesn't find the selector right away:

```
         │ debg TestSubjects.existOrFail(noesaveDashboardSuccess)
         │ debg TestSubjects.exists(noesaveDashboardSuccess)
         │ debg existsByDisplayedByCssSelector [data-test-subj~="noesaveDashboardSuccess"]
         │ debg --- retry.try error: expected false to equal true
         │ debg TestSubjects.existOrFail(noesaveDashboardSuccess)
         │ debg TestSubjects.exists(noesaveDashboardSuccess)
         │ debg existsByDisplayedByCssSelector [data-test-subj~="noesaveDashboardSuccess"]
         │ debg --- retry.try failed again with the same message...
         │ debg TestSubjects.existOrFail(noesaveDashboardSuccess)
         │ debg TestSubjects.exists(noesaveDashboardSuccess)
         │ debg existsByDisplayedByCssSelector [data-test-subj~="noesaveDashboardSuccess"]
         │ debg --- retry.try failed again with the same message...
         │ debg TestSubjects.existOrFail(noesaveDashboardSuccess)
..... (on and on for awhile longer)
```

Don't know what is up with that other jenkins failure you linked to though...

Reply:
OK, that looks right.  I'm going to do a bit of testing on it right now...